test/Confluent.Kafka.IntegrationTests/testconf.json

```diff
@@ -1,5 +1,3 @@
 {
-  "bootstrapServers": "localhost:9092",
-  "singlePartitionTopic": "test-topic-1",
-  "partitionedTopic": "test-topic-2"
+  "bootstrapServers": "localhost:9092"
```

Comment:
I think we should avoid having default brokers for anything, if this happens to be run on the wrong machine it could mess with a production cluster.

Reply:
technically, `testconf.json` shouldn't be checked in yes, I just do it as a convenience. this isn't a regression and it's all about to be replaced with some dockerizing i'm doing at the moment so I don't think needs changing here and now.